thirdparty/civetweb-1.9.1/src/civetweb.c

```diff
@@ -11856,7 +11859,24 @@ ssl_get_client_cert_info(struct mg_connection *conn)
 
 		/* Calculate SHA1 fingerprint and store as a hex string */
 		ulen = 0;
-		ASN1_digest((int (*)())i2d_X509, digest, (char *)cert, buf, &ulen);
+
+		/* ASN1_digest is deprecated. Do the calculation manually,
```

Comment:
is this also a backport?

Reply:
Sorry I missed this response. I was concerned about using custom code in the third party dir, but with that knowledge, with that knowledge it's a non issue. I'll +1 and merge, thanks!